### ISSUE TITLE
Tweaks

### ISSUE THREAD

**Issue by jpike88** [streaka/report-writer-issues#5]
If you select ‘Replenishment system Installed’ - (First question) Area to which Termite Management System Installed. If you select ‘other’ the comment box pops up, but when you put a comment in this text box, the comment is not showing on completed form.

Treatment Details and “Live Termite Activity” and “Nest/Live Termite Control” are selected after the questions “Were termite nests located?” insert a non compulsory type box. 

In Summary when question “Is there a Termite Management Sticker Present? YES add another sticker photo image called “Image of Termite Sticker 2”

Service Details after the question “Details of pest management program and other advice” add a photo option up to 5 photos (if possible). 
Treatments after the Question “Areas of Treatment” add photo option.

**Comment by jpike88** [comment]
[Report Writer Changes Wed 2017 03 01.docx](https://github.com/streaka/report-writer-issues/files/820269/Report.Writer.Changes.Wed.2017.03.01.docx)
